driver_go18_test.go

```diff
@@ -247,12 +293,33 @@ func TestContextCancelExec(t *testing.T) {
 		dbt.mustExec("CREATE TABLE test (v INTEGER)")
 		ctx, cancel := context.WithCancel(context.Background())
 
-		// Delay execution for just a bit until db.ExecContext has begun.
+		var longProcess *dbProcess
+		longQuery := "INSERT INTO test VALUES (SLEEP(1))"
+
+		// Delay execution for just a bit until db.ExecContext has begun and get long process ID
+		defer time.AfterFunc(100*time.Millisecond, func() {
```

Comment:
there is no guarantee that query will be started during 100ms, 
I think it will be better to do this several times with timeout 100ms, during 5 sec for example, (we have sleep 15 sec)
it query does not start for 5 sec, the test should fail.

Reply:
ok